src/plugins/sentryPlugin.ts

```diff
@@ -27,7 +36,13 @@ export const sentryPlugin: ApolloServerPlugin = {
         for (const err of ctx.errors) {
           // Only report internal server errors,
           // all errors extending ApolloError should be user-facing
-          if (err instanceof ApolloError) {
+          if (
+            errorCodes.includes(err.extensions?.code?.toString()) ||
+            //the error-handler is called after sentryPlugin,
+            //so we won't have `code` populated yet
+            //so we are string matching with `NotFoundError` class message prefix
+            err.message.includes('Error - Not Found')
```

Comment:
Might be more robust if we check `err.originalError instanceof NotFoundError`?

Reply:
nice! thanks Kat - this worked 🙌